src/async-wrap.h

```diff
@@ -8,6 +8,8 @@
 
 namespace node {
 
+#define ASYNC_ID_OFFSET 0xA1C
```

Comment:
smalloc is the only other example I can find. We could maintain a central table for classIds. The danger that I am thinking about is if user-space modules need to use use classIds as well and happen to use the same value. It would be nice if one (i.e. user-space or core) could request a classId from the table. That way we can be sure that there won't be collisions.

I am only 'a little bit un-easy', and don't have objections to this PR as is.